research-hub-web/src/app/components/home/home.component.scss

```diff
@@ -1,2 +1,58 @@
-@import "../../../styles";
+@import "../../../typography.scss";
+@import "../../../themes.scss";
+@import "../../../styles.scss";
+@import '../../app.component.scss';
 
+
+/**
+ * Content Container
+ */
+ .content-container {
+    @extend .margin-bottom;
+    padding: 16px 0;
+    width: 95vw;
+    margin: 0 auto;
+    text-align: center;
+}
+
+
+.grey-background {
+    padding-bottom: 40px;
```

Comment:
This and `.teal-background` seem like misleading classnames if they're adding margins... and kinda unnecessary TBH.